src/hooks/index.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('@sveltejs/kit').Handle} */
+export async function handle({ request, resolve }) {
+	const response = await resolve(request, {
+		ssr: false
```

Comment:
Change required due to recent sveltekit update https://github.com/sveltejs/kit/pull/2804. Not sure if this correct for running the site on production though.